lib/base/journaldlogger.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef JOURNALDLOGGER_H
+#define JOURNALDLOGGER_H
+
+#if !defined(_WIN32) && defined(HAVE_SYSTEMD)
+#include "base/i2-base.hpp"
+#include "base/journaldlogger-ti.hpp"
+
+namespace icinga
+{
+
+/**
+ * A logger that logs to systemd journald.
+ *
+ * @ingroup base
+ */
+class JournaldLogger final : public ObjectImpl<JournaldLogger>
+{
+public:
+	DECLARE_OBJECT(JournaldLogger);
+	DECLARE_OBJECTNAME(JournaldLogger);
+
+	static void StaticInitialize();
+	static void StatsFunc(const Dictionary::Ptr& status, const Array::Ptr& perfdata);
+
+	void OnConfigLoaded() override;
+	void ValidateFacility(const Lazy<String>& lvalue, const ValidationUtils& utils) override;
+
+protected:
+	static std::map<String, int> m_FacilityMap;
```

Comment:
Doesn't seem to be used anywhere (anymore).

Reply:
Ack, fixed.